RecoEgamma/EgammaHLTProducers/plugins/EgammaHLTGsfTrackVarProducer.cc

```diff
@@ -67,8 +88,15 @@ class EgammaHLTGsfTrackVarProducer : public edm::global::EDProducer<> {
   const edm::EDPutTokenT<reco::RecoEcalCandidateIsolationMap> validHitsMapPutToken_;
   const edm::EDPutTokenT<reco::RecoEcalCandidateIsolationMap> nLayerITMapPutToken_;
   const edm::EDPutTokenT<reco::RecoEcalCandidateIsolationMap> chi2MapPutToken_;
+  const edm::EDPutTokenT<reco::RecoEcalCandidateIsolationMap> fbremMapPutToken_;
 };
 
+namespace {
+
+  float calRelDelta(float a, float b, float defaultVal = 0) { return b != 0 ? (a - b) / a : defaultVal; }
```

Comment:
Very naively.. should this be `a != 0` since the denominator is `a` (not `b`) ?

Reply:
ah thanks , yes this came in a change earlier when I realised I had fbrem backwards but forgot to change the check, appreciated!